src/EnergyPlus/ReportSizingManager.cc

```diff
@@ -1114,13 +1114,16 @@ namespace ReportSizingManager {
 						}
 					}
 				} else if ( SizingType == HeatingWaterDesCoilLoadUsedForUASizing ) {
-					if ( TermUnitSingDuct || TermUnitPIU || TermUnitIU ) {
```

Comment:
1) not sure why the multiplier is used here.
2) split out SZ reheat coils in case multiplier was needed for other terminal types, can tighten that up later
3) DesMassFlow wasn't used in the equation so deleted that line of code